src/components/DatabaseChanges.tsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { a as A, p as P } from './Mdx';
+
+interface IDatabaseName {
+  product: string;
+}
+
+const DatabaseChanges: React.FC<IDatabaseName> = (props) => {
```

Comment:
@kevcenteno I created this component since the same language was added to each database page. I thought the better move would be to reuse another MDX file like this one: https://github.com/maxmind/dev-site/blob/main/content/geoip/docs/databases/_apis.mdx, but I couldn't figure out how create a variable / placeholder (e.g. database name) in the template MDX file that could then be used in each database MDX file.

Reply:
Nvm, I think this solution might be acceptable given the similarity to`<ZipFileContent product-name="Database Name />` in each of the database MDX files (e.g. https://raw.githubusercontent.com/maxmind/dev-site/main/content/geoip/docs/databases/city-and-country.mdx)